cmd/wallet-sdk-gomobile/api/credentials.go

```diff
@@ -39,6 +41,37 @@ func (v *VerifiableCredential) IssuerID() string {
 	return v.VC.Issuer.ID
 }
 
+// Types returns the types of this VC. At a minimum, one of the types will be "VerifiableCredential".
+// There may be additional more specific credential types as well.
+func (v *VerifiableCredential) Types() *StringArray {
+	return &StringArray{strings: v.VC.Types}
```

Comment:
@sudeshrshetty do we need to pick Types from credentialSubject as well ?

Reply:
Either we can add a function to get `Claim() returning credential subject map` & `ClaimType() which returns credential subject type if found`

Reply:
can be a new PR